projects/igniteui-angular/src/lib/grids/grid/grid-filtering-ui.spec.ts

```diff
@@ -3929,6 +3929,27 @@ describe('IgxGrid - Filtering actions - Excel style filtering', () => {
             [ 'Select All', '20', '254' ],
             [ true, true, true ]);
     }));
+
+    it('Should cascade filter the available filter options.', fakeAsync(() => {
+        fix.detectChanges();
```

Comment:
I believe we do not need this fix.detectChanges()

Reply:
Done.